website/static/js/addProjectPlugin.js

```diff
@@ -7,6 +7,7 @@ require('css/add-project-plugin.css');
 var $ = require('jquery');
 var m = require('mithril');
 var $osf = require('js/osfHelpers');
+var InstitutionImg = require('js/components/institution');
```

Comment:
Minor: This is a bit misleading, because you're not importing the `InstititionImg` component; you're importing the `institution` module. You could do

```js
var institutionComponents = require('js/components/institution');
var CheckableInst = institutionComponents.CheckableInstitution;
```